README.md

```diff
@@ -25,7 +25,7 @@ git clone https://github.com/BenRoe/MMM-SystemStats
 
 Navigate to the new `MMM-SystemStats` folder and install the node dependencies.
 ```
-cd MMM-SystemStats/ && npm install
```

Comment:
in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.